spec_helper.rb

```diff
@@ -48,10 +48,7 @@
   end
 end
 
-dir = "../fixtures/code"
-CODE_LOADING_DIR = defined?(require_relative) ?
-                     File.realpath(dir, __FILE__) :
-                     File.expand_path(dir, __FILE__)
+CODE_LOADING_DIR = File.expand_path "../fixtures/code", __FILE__
```

Comment:
I think the check on require_relative comes from ruby 1.8 support and seems like the difference between `realpath` and `expand_path` is just that realpath ensures the file actually exists, but I might be wrong here…